jha/tests/unit/services/apiService.spec.js

```diff
@@ -0,0 +1,65 @@
+import apiService from "@/services/api-service";
+import dummyData from "@/store/states/enrolment-module-dummy-data.js";
+import { mockFile } from "../../fixtures.js";
+
+apiService._sendPostRequest = jest.fn(() => Promise.resolve(true));
+
+describe("Power Of Attorney", () => {
+  dummyData.hasPowerOfAttorney = true;
+  dummyData.fpcPowerOfAttorneyDocuments = [mockFile];
+
+  const getApplicationAttachments = async (data) => {
+    await apiService.sendApplication(dummyData);
+    let [, , postedFormData] = apiService._sendPostRequest.mock.calls[0];
```

Comment:
I've never seen this before, is this array destructuring and ignoring the first two values?

Reply:
yes, you can just put commas if you only need the nth value

Reply:
And yes that's right The PowerOfAttorney doctype was a request from kevin for maxhub